lib/opensusebasetest.pm

```diff
@@ -119,6 +119,9 @@ done", "binaries-with-missing-libraries.txt", {timeout => 60, noupload => 1});
     script_run 'tar cvvJf problem_detection_logs.tar.xz *';
     upload_logs('problem_detection_logs.tar.xz');
     type_string "popd\n";
+
+    # Detect bsc#1063638
+    record_soft_failure 'bsc#1063638' if (script_run('ps x | grep "btrfs-\(scrub\|balance\|trim\)"') == 0);
```

Comment:
If the post_fail_hook reached *that* far – without failing – then I am sure the processes are gone from the process table. This should rather be something conducted just at the beginning. Just move it to the top, before going to the temporary directory. However, this function is not called in all test modules. I know, the post_fail_hook of the base class does not yet log into a terminal but maybe we need to do that now.